src/VisualStudio/Core/Def/ProjectSystem/VisualStudioWorkspaceImpl.cs

```diff
@@ -209,21 +209,18 @@ public async Task InitializeUIAffinitizedServicesAsync(IAsyncServiceProvider asy
             _memoryListener = memoryListener;
         }
 
-        await _threadingContext.JoinableTaskFactory.SwitchToMainThreadAsync(_threadingContext.DisposalToken);
-
-        // This must be called after the _openFileTracker was assigned; this way we know that a file added from the project system either got checked
-        // in CheckForAddedFileBeingOpenMaybeAsync, or we catch it here.
-        openFileTracker.CheckForOpenFilesThatWeMissed();
-
-        // Switch to a background thread to avoid loading option providers on UI thread (telemetry is reading options).
-        await TaskScheduler.Default;
-
         var logDelta = _globalOptions.GetOption(DiagnosticOptionsStorage.LogTelemetryForBackgroundAnalyzerExecution);
         var telemetryService = (VisualStudioWorkspaceTelemetryService)Services.GetRequiredService<IWorkspaceTelemetryService>();
         telemetryService.InitializeTelemetrySession(telemetrySession, logDelta);
```

Comment:
this was already running in the BG before.  i'm just moving it earlier (where we're already on the BG) instead of having to jump *back* to bg to run this